src/toil_scripts/exome_variant_pipeline/README.md

```diff
@@ -149,3 +159,99 @@ ci-test:
 To run on a distributed AWS cluster, see [CGCloud](https://github.com/BD2KGenomics/cgcloud) for instance provisioning, 
 then run `toil-exome run aws:us-west-2:example-jobstore-bucket --batchSystem=mesos --mesosMaster mesos-master:5050`
 to use the AWS job store and mesos batch system. 
+
+# Methods
+
+## Tools
+
+| Tool     | Version | Description                                                                                       |
+|----------|---------|---------------------------------------------------------------------------------------------------|
+| Samtools | 0.1.19  | Indexes bam files and creates reference index.                                                    |
+| Picard   | 1.95    | Creates sequence dictionary for reference.                                                        |
+| GATK     | 3.5     | Used for GATK preprocessing. Indel realignment (IR) and base quality score recalibration (BQSR).  |
+| Pindel   | 0.2.5b6 | Computes insertions and deletions for each BAM file.                                              |
+| MuSe     | 1.0     | Finds somatic variants from a pair of normal and tumor BAM files.                                 |
+| MuTect   | 1.1.7   | Finds somatic variants from a pair of normal and tumor BAM files.                                 |
+
+## Reference Data
+
+This pipeline is designed to work with HG19 and GRCh37. HG38 / GRCh38 support is in the works. All other input files
+to this pipeline are part of the [GATK bundle](http://gatkforums.broadinstitute.org/gatk/discussion/1213/whats-in-the-resource-bundle-and-how-can-i-get-it).
+
+## Tool options
+
+- Samtools index/faidx are run with default options
+- Picard dictionary is run with default options
+- GATK BQSR is run with default options
+- GATK PR is run with `--emit_original_quals`
```

Comment:
Not related to this PR, but: do we upload the final preprocessed BAMs? If not, you should omit `--emit_original_quals`.